eotsmanager/eotsmanager.go

```diff
@@ -31,6 +31,9 @@ type EOTSManager interface {
 	// or passPhrase is incorrect
 	SignEOTS(uid []byte, chainID []byte, msg []byte, height uint64, passphrase string) (*btcec.ModNScalar, error)
```

Comment:
We can mention that `SignEOTS` should have built-in anti-slashing mechanism to ensure signature for the same height will not be signed twice